### DISCUSSION TITLE
Working Device ZAPO W58L

### DISCUSSION THREAD

**Discussion by mhkanon** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#38] [General]
I using **ZAPO W58L** in **Big Sur 11.2.2**. Perfect working. Check the device details https://www.amazon.in/Adapter-600Mbps-2-4GHz-Wireless-Network/dp/B07R16NLC5

**Comment by chris1111** [comment]
Thanks tomorrow I will ad to the known working list adapter

**Comment by chris1111** [comment]
Done
https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/1af3bc4aaabd6ac1cd96647407cf26918b652096
tanks
